src/scenic/core/object_types.py

```diff
@@ -43,10 +43,6 @@ def __init_subclass__(cls):
 		# find all defaults provided by the class or its superclasses
 		allDefs = collections.defaultdict(list)
 
-		# catch multiple inheritance of Scenic classes
```

Comment:
We found that tests for `carla` and `lgsvl` do not follow the single-inheritance rule and decided to allow multiple subclasses.